addons/im_livechat/report/im_livechat_report_channel.py

```diff
@@ -40,6 +37,15 @@ class Im_LivechatReportChannel(models.Model):
     partner_id = fields.Many2one('res.partner', 'Operator', readonly=True)
     handled_by_bot = fields.Integer("Handled by Bot", readonly=True, aggregator="sum")
     handled_by_agent = fields.Integer("Handled by Agent", readonly=True, aggregator="sum")
+    livechat_failure = fields.Selection(
+        selection=[
+            ("never_answered", "Never Answered"),
+            ("no_one_available", "No one Available"),
+            ("no_failure", "No failure"),
+        ],
+        string="Conversation Failure",
+        readonly=True,
+    )
```

Comment:
In the reporting we're still expecting "outcome" instead with the 4 values of the original spec grouped together, including escalated.

For example to be displayed as a pie chart of outcome.